activerecord/lib/active_record/relation/batches.rb

```diff
@@ -230,9 +230,5 @@ def apply_limits(relation, begin_at, end_at)
       relation = relation.where(table[primary_key].lteq(end_at)) if end_at
       relation
     end
-
-    def batch_order
-      "#{quoted_table_name}.#{quoted_primary_key} ASC"
-    end
```

Comment:
I'd leave the `batch_order` method in place and just define it as:

``` ruby
def batch_order
  { primary_key => :asc }
end
```

There's no need to symbolise the column name.